scripts-setup/setup-git-hooks

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+set -e
+BASEDIR=$(git rev-parse --show-toplevel)
+SCRIPTDIR=$(dirname $0)
+set -e
```

Comment:
Is this a repeat?

Reply:
Yea it's so I don't need to assume `setup-git-hooks` is in `scripts-setup` and let's me run something like `./scripts-setup/setup-git-hooks`.  Let me know if you'd prefer something different.  I could also do `${BASEDIR}/scripts-setup`

Reply:
No, I meant the repeated `set -e`

Reply:
🤦‍♂️ fixed.